src/Database.cpp

```diff
@@ -233,7 +223,7 @@ Database::OpenFile(const char* path)
 		acc->SetLastCheckNumber(acc->LookupLastCheckNumber());
 	}
 
-	gDefaultLocale = GetDefaultLocale();
+	Locale gDefaultLocale;
```

Comment:
Why declaring gDefaultLocale here?

Reply:
It should initialize gDefaultLocale, but I think it is already done in `Database.h` line #124 (`extern Locale gDefaultLocale;`). There was a method that retrieved default settings from the database, not used anymore.
I'll remove it and update the PR